crates/ra_ide_api/src/impls.rs

```diff
@@ -54,10 +63,12 @@ fn impls_for_def(
 
 fn impls_for_trait(
     db: &RootDatabase,
+    position: FilePosition,
```

Comment:
Rather than adding `position` as an argument, can we pack both `poistion` and `node` into `Source<ast::TraitDef>`?

Reply:
Yes, we can, I just keep it written in the same way as `impls_for_def`.